examples/autocomplete-atomic/index.tsx

```diff
@@ -52,32 +79,61 @@ const CityChip: FunctionComponent<any> = (props) => {
     }
 
     return (
-        <Chip 
-            avatar={<Avatar>{value.image}</Avatar>} 
-            label={value.name} 
-            onClick={handleClick} 
+        <Chip
+            avatar={<Avatar>{value.image}</Avatar>}
+            label={value.name}
+            onClick={handleClick}
         />
     )
 }
 
 const AutocompleteAtomic = () => {
+    const classes = useStyles();
     return (
-        <MUIRichTextEditor 
+        <MUIRichTextEditor
+            classes={{
+                autocomplete: classes.autocomplete,
+            }}
             label="Try typing '/mexico'..."
             onSave={save}
             customControls={[
                 {
                     name: "my-city",
                     type: "atomic",
                     atomicComponent: CityChip
-                },
+                }
             ]}
             autocomplete={{
                 strategies: [
                     {
                         items: cities,
                         triggerChar: "/",
-                        atomicBlockName: "my-city"
+                        atomicBlockName: "my-city",
+                        minSearchChars: 1,
+                    },
+                    {
+                        items: shortcutBullet,
+                        triggerChar: "*",
+                        minSearchChars: 1,
+                        handleAutoComplete: (editorState: EditorState, selectionState: SelectionState, value: string): EditorState => {
```

Comment:
Extract this function into a standalone function

Reply:
Addressed